cromwell_tools/cli.py

```diff
@@ -106,6 +111,11 @@ def add_auth_args(subcommand_parser):
 
 
 # this should just getattr from CromwellAPI and call the func with args.
+# TODO: refactor this module into class-based parsers
 def main(arguments=None):
     command, args = parser(arguments)
-    print(command(**args))
+    API_result = command(**args)
+    if isinstance(API_result, requests.Response):
+        print(API_result.text)
+    else:
+        print(API_result)
```

Comment:
This is just a workaround for improving the CLI stdouts, in the next PR, we shall refactor the CLI into a well-designed class!